cmd/get_secret.go

```diff
@@ -86,6 +92,21 @@ func (s *SecretDescription) PrettyPrintSecretDesc() {
 	}
 }
 
+// ExportSecret prints out secret in exported format.
+func (s *SecretDescription) ExportSecret() {
+	// Trim away container from name if found..
+
+	name := strings.TrimSuffix(s.Name, "-"+s.Container)
+
+	var d []string
+	for k, v := range s.Data {
+		d = append(d, fmt.Sprintf("--data '%v=%v'", string(k), string(v)))
+	}
+	data := strings.Join(d, " ")
+	out := fmt.Sprintf("kubectl tbac create secret %v --namespace %v %v", name, s.Namespace, data)
```

Comment:
Doesn't this print out the command rather than the actual secret?

Reply:
Yes, it prints out the command to create the secret as if it didn't exist. See the example above.
Then the user can change the command how they like. So they can update the namespace to sandbox for example, change some data values and change the name of the secret.

Reply:
Ah, OK, I thought it exported the actual secret resource but I got it now 👍